locksmith/src/routes/user.js

```diff
@@ -0,0 +1,10 @@
+let express = require('express')
+
+let router = express.Router()
+/* eslint-disable import/no-unresolved*/
```

Comment:
Could you explain what this is?

Reply:
this is referencing the result of a typescript file that will exist, when build. As there actually isn't a javascript file for this controller prior to being built the linter raises an error

Reply:
awww, this is weird. @cnasc is there a way to deal with that in a better way?